homeassistant/components/mikrotik/hub.py

```diff
@@ -54,6 +54,11 @@ def name(self):
         """Return device name."""
         return self._params.get("host-name", self.mac)
 
+    @property
+    def ip_address(self):
+        """Return device primary ip address."""
+        return self._params.get("address")
```

Comment:
In the `attrs` device property `"active-address"` is used. What is the difference?

Also please deduplicate the state attributes returned in `device_state_attributes`. It seems mac address and ip address are returned there as well.

Reply:
From digging though the code, it appears `active-address` is the address the device has when the dhcp request has been answered so it could be empty. `address` is the address that is remembered in the dhcp table. Since we check state to make sure the device is `home`, it seemed like `address` was better since it won't ever be unset.

Reply:
Duplicate attributes removed here https://github.com/home-assistant/core/pull/45255